docs/user/metal/Troubleshooting_Ipv6.md

```diff
@@ -0,0 +1,277 @@
+# IPv6 Disconnected Install Troubleshooting
```

Comment:
`docs/user/metal/Troubleshooting_Ipv6.md` -> `docs/user/metal/troubleshooting_ipv6.md`

It has content like 
```
## Create a Disconnected Registry

# BootstrapVM cannot download RHCOS image

## Bootstrap VM cannot boot up my nodes

... and more
```

which talk about general troubleshooting of baremetal ipi  and don't seem to be related to ipv6. So i think we should split these into generic and special ipv6 files.